api/src/main/java/org/openmrs/module/muzimafingerPrint/api/db/hibernate/HibernateMuzimaFingerprintDAO.java

```diff
@@ -29,12 +26,14 @@ public MuzimaFingerprintDAOImpl(SessionFactory factory) {
     }
 
     @Override
-    public void saveMuzimaFingerprint(MuzimaFingerprint fingerprint) {
+    public MuzimaFingerprint saveMuzimaFingerprint(MuzimaFingerprint fingerprint) {
         try{
-        session().saveOrUpdate(fingerprint);}
-        catch (Exception e){
+            session().saveOrUpdate(fingerprint);
+            return fingerprint;
+        } catch (Exception e){
             e.printStackTrace();
```

Comment:
I think this is not your code, but I think it's more preferable way to use logging framework. See other code in OpenMRS on how they use the logging. Please update them in the future.

I think it's usually like the following:
```Logger log = LogFactory.getLogger(Class.class)```